improving-app/riddl
# IA-123: Upgrade Improving-app/riddl to use riddlc release 0.20.0

Primary Reviewer: @reid-spencer 
Secondary Reviewer if necessary: @JamesYopp 

### Issue:
- 0.19.0 riddl introduces changes to the syntax of streaming. Changes must be made to appropriately validate in this version for the Improving App riddl
- 0.20.0 riddl introduces record for the states. Changes must be made to appropriately validate this version for the Improving App riddl

### Changes:
- remove `from ...` from affected outlets
- add `of ^...` to all states

### Testing:
- ensure local machine uses 0.20.0 riddl
- run validation stage, it should. pass

### TODO for this PR:
- please update the github actions to use 0.19.0 riddl rather than 0.18.0
